common/src/main/java/dev/ftb/mods/ftbquests/gui/quests/ViewQuestPanel.java

```diff
@@ -437,7 +437,7 @@ private void addDescriptionText(boolean canEdit, Component subtitle) {
 	}
 
 	private void addButtonBar(boolean canEdit) {
-		// button bar has page navigation buttons for multi-page text, and the Add button in edit mode
+		// button bar hasVisibleQuest page navigation buttons for multi-page text, and the Add button in edit mode
```

Comment:
search and replace? :smile:

Reply:
Damn it! I new I would miss one. That's intellij's stupid "You renamed something common so let me go refactor all the comments"... I'll fix it in a bit